### PR TITLE
chore(app/workspace): use fileSystem to read files

### LINKED ISSUE #1002: Use filesystem when restoring workspace
Workspace [doesn't use FileSystem](https://github.com/camunda/camunda-modeler/blob/next/app/lib/workspace.js#L25) when restoring.

### PR DESCRIPTION
**Pull request by nikku**
Closes #1002

### CODE REVIEW

File: app/lib/index.js [hunk 1]
```diff
@@ -68,8 +68,13 @@ global.modelerDirectory = appPath;
 
 var menu = new Menu(process.platform);
 
+// bootstrap filesystem
+var fileSystem = new FileSystem({
+  dialog: dialog
```
Comment (philippfromme) [inline app/lib/index.js]: File system doesn't depend on dialog anymore.

Comment (nikku) [reply app/lib/index.js]: Separate issue?

Comment (nikku) [reply app/lib/index.js]: Or simple quick fix?

Comment (philippfromme) [reply app/lib/index.js]: I'll fix it. 👍

**Pull request merged**